src/enki_files_valiadtor.py

```diff
@@ -1,186 +1,34 @@
 #!/usr/bin/env/ python3
```

Comment:
Due to the extra forward slash after `env`, this interpreter line is invalid and would result in the following error if actually executed:

```
bash: ./enki_files_validtor.py: /usr/bin/env/: bad interpreter: Not a directory

```

The same problem occurs in `enki_msg.py`.

In addition, the use of `/usr/bin/env` is inconsistent among the files as most use `/usr/bin/python3` directly. Considering none of the files other than `enki.py` have executable code or are executable files, I would however suggest removing the interpreter directive from them.